packages/nimble-components/src/table-column/icon/group-header-view/index.ts

```diff
@@ -55,8 +56,15 @@ export class TableColumnIconGroupHeaderView
         } else if (mappingConfig instanceof MappingSpinnerConfig) {
             this.text = mappingConfig.text ?? '';
             this.visualizationTemplate = mappingConfig.spinnerTemplate;
+        } else if (mappingConfig instanceof MappingTextConfig) {
+            this.text = mappingConfig.text ?? '';
         }
     }
+
+    private resetState(): void {
+        this.text = '';
```

Comment:
wanna reset the severity state too? just all the state pulled from the mapping config